frontend/app/components/mapbox/filter-features.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+
+export default class MapboxFilterFeaturesComponent extends Component {
+  map = {}
+  layerId = null;
+  filterById = "id";
+  // map will be restricted to this array of features
+  featureIds = null;
+
+  @computed('featureIds', 'filterById')
+  get filterToFeatures(){
```

Comment:
I'm not sure I understand the naming of this function. To my mind what this does is update the filter when featureIds is updated

(also, are we overriding computes again here to trigger  an action? Should the computed property just be the filter itself? and then in `didUpdateAttrs()` we just call `instance.setFilter(this.layerId, this.filter)`?

Reply:
And same comment as above.... I should definitely write it exactly as you described.